File: Go/fizzbuzz3.go
package main

import (
	"fmt"
)

const LIMIT = 100

func main() {
	for i := 1; i <= LIMIT; i++ {
		switch i % 15 {
		case 0:
			fmt.Print("FizzBuzz")
		case 3, 6, 9, 12:
			fmt.Print("Fizz")
		case 5, 10:
			fmt.Print("Buzz")
		default:
			fmt.Print(i)
		}
		if i < LIMIT {
			fmt.Print(", ")
		}
	}
	fmt.Println()
}

File: Julia/fizzbuzz3.jl
const limit = 100

foreach(1:limit) do i
    if i % 15 == 0
        print("FizzBuzz")
    elseif i % 3 == 0
        print("Fizz")
    elseif i % 5 == 0
        print("Buzz")
    else
        print(i)
    end
    if i < limit
        print(", ")
    end
end

println()
